bin/main.ml

```diff
@@ -10,7 +10,7 @@ let to_exit_code : (unit, string) result Term.t -> Cmd.Exit.code Term.t =
 
 let lint (changed_pkgs, new_pkgs) local_repo_dir =
   match local_repo_dir with
-  | None -> Error "No opam repository directory specified."
+  | None -> failwith "TODO: default to using the opam repository"
```

Comment:
The `local_opam_repo_term` is a common option for both lint and revdeps functionality. I'm not sure what the "default" opam repository should be, when no repository option has been provided. Maybe we should use a different option for the `lint` subcommand and require an opam repository directory.

Reply:
My thought was that we should default to the official opam repo if this is not overriden. I don't think this creates any conflicts with the `local_opam_repo_term` tho, since this function is defined independently from that `_term`, and we can always apply it to a different CLI flag.

I'll punt on this question now, and we can figure out the preferred behavior here in a followup. Thanks for the review!